utils/nctl/sh/scenarios-upgrades/upgrade_scenario_09.sh

```diff
@@ -48,7 +48,7 @@ function _main()
     INITIAL_PROTOCOL_VERSION=$(get_node_protocol_version 1)
     # Establish consistent activation point for use later.
     ACTIVATION_POINT="$(($(get_chain_era) + NCTL_DEFAULT_ERA_ACTIVATION_OFFSET))"
-    PRE_UPGRADE_BLOCK_HASH="$($(get_path_to_client) get-block --node-address "$(get_node_address_rpc '2')" | jq -r '.result.block.hash')"
+    PRE_UPGRADE_BLOCK_HASH="$(get_block_v1 "2" | jq -r '.hash')"
```

Comment:
Maybe add a TODO here to change in the future. Now this will work since we're upgrading from 1.x to 2.x but since this test is rather generic, in the future we'll want to update from 2.a to 2.b which will cause a problem yet again.

But I think there are many other situations like this in the scripts so don't think it's worth the bother to fix it now, just take a note.

Reply:
done